fdbserver/worker.actor.cpp

```diff
@@ -2267,7 +2267,7 @@ ACTOR Future<Void> workerServer(Reference<IClusterConnectionRecord> connRecord,
 					CODE_PROBE(true, "Recruited while already a blob migrator.");
 				} else {
 					startRole(Role::BLOB_MIGRATOR, recruited.id(), interf.id());
-					DUMPTOKEN(recruited.waitFailure);
+					DUMPTOKEN(recruited.haltBlobMigrator);
```

Comment:
IMO we should also dump the migrator's storage server interface endpoints, since it's basically just a proxy for that

Reply:
fixed. we should dump storage server interface endpoints.